modules/about-certificate-management.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-a-custom-pki.adoc
+
+[id="about-certificate-management_{context}"]
+= About certificate management
+
+Once a custom certificate is set, the network operator merges the user-provided
+and system CA certificates into a single bundle and injects the merged bundle
```

Comment:
What is meant by "Once a custom certificate is set". It might be better to be explicit stating how a custom ca cert is added to the cluster (i.e. configmap). I would also consider breaking this into 2 or more sentences.